types/web3/utils.d.ts

```diff
@@ -2,13 +2,18 @@ import BigNumber = require("bn.js");
 import * as us from "underscore";
 
 type Unit =
+    | "noether"
+    | "wei"
     | "kwei"
+    | "Kwei"
     | "femtoether"
     | "babbage"
     | "mwei"
+    | "Mwei"
     | "picoether"
     | "lovelace"
-    | "qwei"
+    | "gwei"
+    | "Gwei"
```

Comment:
i suggest to not pollute the type's namespace with same strings but different case in it.

Reply:
So remove line 16?

Reply:
I would keep it in. The "pollution" is done upstream: https://web3js.readthedocs.io/en/1.0/web3-utils.html#id76

Reply:
The reason why I decided to add different capitalisation is because otherwise it can show semantic compile-time errors on otherwise 100% valid code. As noted by @Levino, this is the way it's done in the `web3` lib itself. If/when web3 maintainers remove duplicate strings with different letter case, I'll be happy to modify the list to reflect changes.